pages/index.js

```diff
@@ -1,7 +1,9 @@
+import ServiceList from "../components/ServiceList/index";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <ServiceList />
+    </>
```

Comment:
**It's not necessary using React's Fragment syntax `<>...</>` 🧹**
Fragments 💎 are useful when you want to return multiple sibling elements from a component.

```suggestion
      <ServiceList />
```